TrueSDK/External/TCTrueSDK.m

```diff
@@ -41,6 +42,11 @@ - (void)setupWithAppKey:(nonnull NSString *)appKey
 {
     self.appKey = appKey;
     self.appLink = appLink;
+    self.titleType = TitleTypeDefault;
+}
+
+- (void)setTitleType:(enum TitleType)titleType {
+    self.titleType = titleType;
```

Comment:
I think this is recursive, why do you need a setter?

Reply:
self.titleType = titleType; call setTitleType

Reply:
Done.